src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -25,32 +25,6 @@ public class InvokeRestMethodCommand : WebRequestPSCmdlet
     {
         #region Parameters
 
-        /// <summary>
-        /// Gets or sets the parameter Method.
-        /// </summary>
-        [Parameter(ParameterSetName = "StandardMethod")]
-        [Parameter(ParameterSetName = "StandardMethodNoProxy")]
-        public override WebRequestMethod Method
```

Comment:
It seems we have the same in InvokeWebMethodCommand.Common.cs

Reply:
I kept the one in InvokeWebRequestMethodCommand.Common.cs, this one was redundant